src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import Label from '../label';
+import { StyledRadioButtonGroup } from './radio-button.style';
+
+function initialTabIndex(index) {
+  return index ? -1 : 0;
+}
+
+function checkedTabIndex(checked) {
+  return checked ? 0 : -1;
+}
+
+const RadioButtonGroup = (props) => {
+  const { children, groupName, label } = props;
+  const [selected, setSelected] = useState(undefined);
```

Comment:
The name `selected` sounds like a boolean to me (similar to how a variable named `checked` is usually a boolean).

Maybe `selectedValue` would be a clearer name for this state variable?